packages/iov-tendermint-rpc/src/client.spec.ts

```diff
@@ -92,6 +102,18 @@ function defaultTestSuite(rpcFactory: () => RpcClient, adaptor: Adaptor): void {
     client.disconnect();
   });
 
+  it("gets the same tx hash from backend as calculated locally", async () => {
```

Comment:
Yes, this changed between versions, right?

Reply:
Correct. I made hash test in the test abolve less specific (was fixed to length 20) and here I test hasher against backend. It is 20/32 bytes depending on the version but the test is generic and supports both